src/documents/tests/test_tasks.py

```diff
@@ -89,6 +91,112 @@ def test_train_classifier(self):
         mtime3 = os.stat(settings.MODEL_FILE).st_mtime
         self.assertNotEqual(mtime2, mtime3)
 
+    def test_barcode_reader(self):
+        test_file = os.path.join(
+            os.path.dirname(__file__),
+            "samples",
+            "patch-code-t.pbm",
+        )
+        img = Image.open(test_file)
+        separator_barcode = str(settings.CONSUMER_BARCODE_STRING)
+        self.assertEqual(tasks.barcode_reader(img), [separator_barcode])
+
+    def test_barcode_reader2(self):
```

Comment:
I can get what the test is doing if I look at the input file, but it would be easier if the naming was descriptive for the added test cases.

test_barcode_reader_no_barcode_expected